scripts/docker/config/php/hide-deprecation.ini

```diff
@@ -0,0 +1 @@
+error_reporting = E_ALL & ~E_NOTICE & ~E_DEPRECATED
```

Comment:
For dev purposes, do we actually want to hide deprecation notices? Surely they're useful..

Reply:
Hmm, that's a good point. I think there were some deprecation warnings that broke Flarum with 8.1 at the time I actually added this locally.

Maybe keep the file, but comment it out in the Dockerfile so it's not enabled unless needed for some reason?

Reply:
That seems like a good compromise :)